1-org/README.md

```diff
@@ -163,6 +163,15 @@ Run `terraform output cloudbuild_project_id` in the `0-bootstrap` folder to see
    if [ ! -z "${ACCESS_CONTEXT_MANAGER_ID}" ]; then sed -i "s=//create_access_context_manager_access_policy=create_access_context_manager_access_policy=" ./envs/shared/terraform.tfvars; fi
    ```
 
+1. **(Optional)** If you are deploying The Terraform Example Foundation in a Folder instead of an Organization, you maybe want to add a suffix in tag keys to avoid collision. The script below enables a terraform variable to achieve this:
```

Comment:
Since this is optional and folder deployment mode is [primarily for testing](https://github.com/terraform-google-modules/terraform-example-foundation/blob/7b0d0a6dfa0eb99b05f913b4346304cd5a44f6e2/0-bootstrap/terraform.example.tfvars#L31-L33), I am leaning towards not having this in the readme as it is already documented in https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/1-org/envs/shared/terraform.example.tfvars#L36-L38

Reply:
Done